doc/tutorial/basics/6_expressions/40_listcomp.txtar

```diff
@@ -11,7 +11,7 @@ Lists can be created with list comprehensions.
 The example shows the use of `for` loops and `if` guards.
 
 -- listcomp.cue --
-[ for x in #items if x rem 2 == 0 { x*x } ]
+[for x in #items if x rem 2 == 0 { x*x } ]
```

Comment:
I assume some of these changes were done via search-and-replace, because I see you removed the leading space but not the trailing space.

I would be consistent in replacing both :)